response-body-view.html

```diff
@@ -156,7 +177,7 @@
     <paper-icon-button class="close-preview" title="Close the preview" icon="arc:close" on-tap="closePreview" hidden$="[[!contentPreview]]"></paper-icon-button>
 
     <paper-dialog id="saveDialog" on-iron-overlay-closed="_downloadDialogClose">
-      <h2>Save to file</h2>
+      <h2 class="dialog-title">Save to file</h2>
       <div>
         <p>Your file is ready to download. Click the icon below to save it to your local drive.</p>
         <div class="donnload-section">
```

Comment:
Remove this part. It doesn't make sense to have both button and action icon.

Reply:
@jarrodek As I see, you've updated the repo and closed my PR so I don't need to update it.